readme.md

```diff
@@ -1,7 +1,7 @@
 # CCS specifications
 
-Website: https://ccs-specs.icpc.io
-Github: https://github.com/icpc/ccs-specs
+Website: <https://ccs-specs.icpc.io>  
```

Comment:
Trailing space.

Reply:
That's what I stated in the PR description: this is required for a newline to be there.

Reply:
Sorry, nevermind. Although ugly that you have to do this.